### PR TITLE
Simplify the logic of creating extensions

### PR DESCRIPTION
**Pull request by inksnw**
Simplify the logic of creating extensions

### CODE REVIEW

File: pkg/extension/templatesapp/charts/base/files/.DS_Store
Comment (iawia002) [inline pkg/extension/templatesapp/charts/base/files/.DS_Store:1]: `.DS_Store` should not be committed, and do we need this empty directory to exist in the repo?

File: pkg/extension/templatesapp/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace \
+  --set controller.service.type=NodePort \
+  --set controller.service.nodePorts.http=30888
+```
```
Comment (iawia002) [inline pkg/extension/templatesapp/README.md:7]: What are these? They seem irrelevant to the contents of this directory, and this is inconsistent with the content in `README_zh`.

File: pkg/extension/templatessimple/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+```bash
+helm upgrade --install ingress-nginx ingress-nginx \
+  --repo https://kubernetes.github.io/ingress-nginx \
+  --namespace ingress-nginx --create-namespace \
+  --set controller.service.type=NodePort \
+  --set controller.service.nodePorts.http=30888
+```
```
Comment (iawia002) [inline pkg/extension/templatessimple/README.md:7]: ditto

File: pkg/extension/templatesapp/README.md (hunk 1 above)
Comment (inksnw) [reply pkg/extension/templatesapp/README.md]: Done

File: pkg/extension/templatessimple/README.md (hunk 2 above)
Comment (inksnw) [reply pkg/extension/templatessimple/README.md]: Done

**Review by iawia002 — APPROVED** [review]
